backend/curriculum_tracking/serializers.py

```diff
@@ -625,4 +625,21 @@ class Meta:
             "project_cards_total_count",
             "cards_in_complete_column_total_count",
             "project_cards_in_complete_column_total_count",
-        ]
\ No newline at end of file
+        ]
+
+class ReviewTrustSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = models.ReviewTrust
+        fields = [
+            "id",
+            "content_item",
+            "content_item_title",
+            "flavours",
+            "user",
+        ]
+
+    content_item_title = serializers.SerializerMethodField('get_content_item_title')
+    flavours = serializers.CharField(help_text="comma separated list of flavours")
```

Comment:
This implies that someone will be filling in a form to add some flavours.

Can you rather look at how AgileCardSerialiser exposes flavours? 

I think there is some flavour technical debt that may have mislead you a bit

Reply:
The AgileCardSerializer just has "flavour_names" in the fields list.  I changed things so that it looks like the AgileCardSerializer but I am not sure if the change is what you wanted?

Reply:
Individually my tests pass but when I try to run the class inside which my tests exist I get the following:

AssertionError: False is not true : flavour_names = []. The value must be truthy

I have a hack to get around the problem but I am fairly sure it won't be allowed.  What should I do?